### PR TITLE
Adds matomo

### PR DESCRIPTION
**Pull request by christinach**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -35,8 +35,9 @@
     <!--[if lt IE 9]>
       <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
-
+
```
Comment (tampakis) [inline app/views/layouts/application.html.erb]: Get rid of this line

**Comment by christinach** [comment]
I removed the package-lock.json. It was created when I tested the npm matomo package. 
I'm in favor of adding Webpack to control the js and css files. It would be much easier to noParse files that we don't need in specific environments.

**Comment by christinach** [comment]
Right now for testing purposes will track the traffic both in catalog-staging and catalog.

**Comment by christinach** [comment]
colses #1263

**Pull request merged**